### ISSUE TITLE
1x1 Symbol

### ISSUE THREAD

**Issue by jimb-cc** [premnirmal/StockTicker#206]
It would be great to be able to get a 1x1 sized widget that was specially designed to work at that size rather than scaling down the graph.  It would show the symbol name, sparkline and current value.  The example below is from IOS which has a 1x1 just like this, but I can't see one on any android app anywhere.
<img src="https://user-images.githubusercontent.com/33234558/111180279-c9b2c880-85a4-11eb-80d7-c190bf0a784d.png" width="100" height="100" />

**Comment by Kaneki-x** [comment]
> It would be great to be able to get a 1x1 sized widget that was specially designed to work at that size rather than scaling down the graph. It would show the symbol name, sparkline and current value. The example below is from IOS which has a 1x1 just like this, but I can't see one on any android app anywhere.
> <img alt="" width="100" height="100" src="https://user-images.githubusercontent.com/33234558/111180279-c9b2c880-85a4-11eb-80d7-c190bf0a784d.png">
Yes, couldn't agree more, I can't find an app on android that is of the same caliber as the ios built-in stock app widget
